alf/algorithms/actor_critic_algorithm.py

```diff
@@ -134,6 +140,33 @@ def __init__(self,
             loss = loss_class(debug_summaries=debug_summaries)
         self._loss = loss
 
+        # The following checkpoint loading hook handles the case when value
+        # network is not constructed. In this case the value network paramters
+        # present in the checkpoint should be ignored.
+        def _deployment_hook(state_dict, prefix: str, unused_loacl_metadata,
+                             unused_strict, unused_missing_keys,
+                             unused_unexpected_keys, unused_error_msgs):
+            """Preprocess the state_dict to delete incompatible keys.
+
+            In pure privileged mode (i.e. not standard encoder) or pure
+            adaptation mode (i.e. no privileged encoder), this hook will exclude
+            the corresponding keys from the state dict when loading the
+            checkpoint, explicitly avoid crashing caused by such
+            incompatibility.
```

Comment:
This comment (e.g. privileged mode) does not make sense without broader context.

Reply:
The comment was move up and didn't remove this. Fixed.